backend/init-mongo.dockerfile

```diff
@@ -2,7 +2,7 @@
 # speeds up development, since we can rebuild and create new containers without
 # overwriting every time.
 
-FROM python:3.11.5
+FROM python:3.12.3-slim
```

Comment:
Nit: you can use alpine here to make it a bit less intensive. Sometimes that fucks up arm users, so make sure you test it on an arm machine too.

Reply:
oooo ya i looked into this a while back, and read it actually kinda sucks for anything with wheels https://www.reddit.com/r/docker/comments/g5hb93/alpine_vs_pythonslim_for_deploying_python_data/ (and doesnt that include levenstein?)


(init-mongo is dead btw so this would only really be for ci-backend or whatever)